zerver/openapi/zulip.yaml

```diff
@@ -10077,12 +10077,39 @@ paths:
       description: |
         Permanently delete a message.
 
-        This API corresponds to the
-        [delete a message completely][delete-completely] feature documented in
-        the Zulip help center.
+        This API corresponds to the [delete a message completely][delete-completely]
+        feature documented in the Zulip help center.
+
+        A user must be able to access the content of a message in order to delete it.
+        See [channel permissions](/help/channel-permissions) for more information
+        about content access for channel messages. For direct messages, the user
+        must have received or sent the direct message to have content access.
```

Comment:
This is good information, but I wonder if a better home for it might be in the help center, say at https://zulip.com/help/restrict-message-editing-and-deletion#configure-message-deletion-permissions .

Reply:
Yeah moving it there is probably the right idea for this detail.